src/models/cost_functions/HydroGenerationCost.jl

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
Would it help users to make cost_functions a submodule? Maybe it helps searchability.